stdlib/tools/swift-reflection-test/swift-reflection-test.c

```diff
@@ -741,26 +799,24 @@ int reflectAsyncTask(SwiftReflectionContextRef RC,
   }
 
   printf("\n\n");
-  PipeMemoryReader_sendDoneMessage(&Pipe);
+  PipeMemoryReader_sendDoneMessage(Reader);
   fflush(stdout);
   return 1;
 }
 
 
-int doDumpHeapInstance(const char *BinaryFilename) {
-  PipeMemoryReader Pipe = createPipeMemoryReader();
-
+int doDumpHeapInstance(const char *BinaryFilename, PipeMemoryReader *Reader) {
```

Comment:
`doDumpHeapInstance` now accepts a pointer to the `PipeMemoryReader` instead of allocating one locally.